config/drupal/block.block.custompardotform.yml

```diff
@@ -17,19 +17,23 @@ settings:
   label: 'Custom Pardot form'
   provider: bglobal_pardot
   label_display: visible
-  action_url: 'https://go.pardot.com/l/102272/2016-11-04/2klckm'
-  success_location: 'https://extension.berkeley.edu/international/asdasd'
-  error_location: 'https://extension.berkeley.edu/international/'
-  field_list:
-    firstname: firstname
-    lastname: lastname
-    email: email
-    question: question
-    opt_in_contact: opt_in_contact
-  submit_label: 'Apply Now'
-  success_message: 'Success message...'
-  tracking_fields:
-    full_path: full_path
-    page_title: page_title
-    program_name: 0
+  # What does adding the fields here do? 
```

Comment:
This is the stored configuration of the default block. These values are set when you import configuration.

https://support.acquia.com/hc/en-us/articles/360048082073-Configuration-management-in-Drupal-8

Reply:
When we use this block as a field on a node, these values can be overwritten for that specific instance.

Reply:
These lines need to be restored and the comments removed. You can do that by re-exporting this configuration through the UI or with `drush cex` -- just be sure to only check in the change to this file.

Reply:
I restored that part I had commented out.